wazo_agentd/service/handler/on_queue.py

```diff
@@ -59,10 +67,22 @@ def handle_on_agent_unpaused(self, msg):
             'Executing on agent unpaused command (MemberName %s)', msg['MemberName']
         )
         pause_info = self._get_pause_info(msg)
+        if not pause_info:
+            logger.info(
```

Comment:
Avoid spamming admin with normal behavior
```suggestion
            logger.debug(
```